.gitignore

```
@@ -11,6 +11,7 @@ build_release_with_debug_info
 out
 out.py
 .kdev4
+*.kdev4
```

The `.kdev4` should be covered by `*.kdev4`. Maybe check first.

```suggestion
*.kdev4
```

---

it is indeed:)